charts/datadog/templates/cluster-agent-deployment.yaml

```diff
@@ -181,6 +181,10 @@ spec:
           - name: DD_ORCHESTRATOR_EXPLORER_ENABLED
             value: "true"
           {{- end }}
+          {{- if .Values.datadog.orchestratorExplorer.container_scrubbing.enabled }}
```

Comment:
this doesn't work actually, we should check the value, not just if it's set. If i set it to false it still gets enabled

Reply:
same issue with `{{- if .Values.datadog.orchestratorExplorer.enabled }}` above (my fault!). Could you fix both please?

Reply:
i think we can just always set the env vars, but set them to the value of the options from values.yaml (since they're mandatory)